src/app/message/message.css

```diff
@@ -40,6 +40,11 @@ h3 a {
   min-width: 40px;
   min-height: 40px;
   margin-top: 15px;
+  border: 3px solid #fa824c;
```

Comment:
Having this coral color by default doesn't look great imho. I'd say we should set it to `transparent` and just set a border for recent meditators. Additionally I wouldn't add it to normal chat messages. Maybe questions only?